src/common/src/buffer/bitmap.rs

```diff
@@ -161,9 +174,12 @@ impl BitmapBuilder {
     }
 
     pub fn finish(self) -> Bitmap {
+        if self.len == self.count_ones {
+            return Bitmap::ones(self.len);
+        }
         Bitmap {
             num_bits: self.len(),
-            bits: self.data.into(),
+            bits: Some(self.data.into()),
```

Comment:
With bitmap builder it seems we won't ever construct the `None` case?

Reply:
Should we provide some interface to do so? Or we will always construct Bitmap::ones / Bitmap::zeros directly?

Reply:
Thanks for the catch! has fixed.